web/src/pages/Profile/Votes/index.tsx

```
@@ -0,0 +1,69 @@
+
+  const handlePageChange = (newPage: number) => {
+    navigate(`${location}/${newPage}/${order}/${filter}?${searchParams.toString()}`);
+  };
+
+  return (
+    <Container>
+      <StyledTitle>Votes</StyledTitle>
+      <StatsAndFilters totalVotes={10} votesPending={1} resolvedVotes={5} />
+      <VotesCardContainer>
+        {Array.from({ length: 5 }).map((_, index) => (
+          <VoteCard />
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)